app/components/inventory/rank-select.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+import SelectValues from 'hospitalrun/utils/select-values';
+
+const c = Ember.computed;
+
+export default Ember.Component.extend({
+  label: 'Rank',
+  rankOptions: [],
```

Comment:
this looks like instance state, it should likely be instantiated in the initializer, or as a CP.

Placing instance state of the prototype can often lead to unexpected behavior and leaks)